cmd/state-offline-installer/install.go

```diff
@@ -236,21 +232,52 @@ func (r *runner) Run(params *Params) (rerr error) {
 	}
 
 	/* Configure Environment */
-	if err := r.configureEnvironment(params.path, asrt); err != nil {
+	if err := r.configureEnvironment(targetPath, asrt); err != nil {
 		return errs.Wrap(err, "Could not configure environment")
 	}
 
 	r.analytics.Event(ac.CatOfflineInstaller, ac.ActOfflineInstallerSuccess, installerDimensions)
 
-	r.out.Print("Runtime installation completed.")
+	r.out.Print(fmt.Sprintf("Installation complete. Installed to [ACTIONABLE]%s[/RESET].", targetPath))
```

Comment:
I've avoided it as I want to minimize cross-contamination between the state tool and the offline installer because the latter will be moving into its own repo soon.

Reply:
In that case we may want to remove this: https://github.com/ActiveState/cli/pull/2156/files#diff-757a5141926dc28db6117046fb2e33a689bcc4e634e60c203a97279e554b5539R110

Reply:
Can't, input errors and locales are married concepts. But this relies on the fallback mechanic for getting the locale value so it's negligible.